index.js

```diff
@@ -19,11 +19,21 @@ var nodeStatusCodes = require('node-status-codes');
 var isPlainObj = require('is-plain-obj');
 var parseJson = require('parse-json');
 
+function backoff(iter) {
+	var noise = Math.random() * 1000;
+	return (1 << iter) * 1000 + noise;
+}
+
+function isNetworkError(err) {
+	return err.code === 'EAI_AGAIN' || err.code === 'ETIMEDOUT' || err.code === 'ESOCKETTIMEDOUT';
```

Comment:
`ETIMEDOUT` and `ESOCKETTIMEDOUT` is from [timed-out](https://github.com/floatdrop/timed-out), `EAI_AGAIN` from issue. Is there definitive list of those?